ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -722,6 +722,21 @@ static void setPreVote(RaftProperties properties, boolean enablePreVote) {
     }
   }
 
+  interface ReadOnly {
+    String PREFIX = RaftServerConfigKeys.PREFIX
+            + "." + JavaUtils.getClassSimpleName(ReadOnly.class).toLowerCase();
+
```

Comment:
Add a conf to enable/disable linearizable read.

Reply:
The same as above.